src/main/resources/assets/computercraft/lua/rom/motd.txt

```diff
@@ -3,3 +3,13 @@ View the documentation at https://wiki.computercraft.cc
 Visit the forum at https://forums.computercraft.cc
 You can disable these messages by running "set motd.enable false"
 You can create directories with "mkdir".
+Want to see hidden files? Run "set list.show_hidden true".
+Run "list","ls" or "dir to see all files in a directory.
```

Comment:
Maybe " "list" (or "ls" for short) to see all files in a directory." Listing all aliases may be a little too much.